thinc/util.py

```diff
@@ -47,10 +47,17 @@ def get_torch_default_device() -> "torch.device":
 
 
 def get_array_module(arr):  # pragma: no cover
-    if is_cupy_array(arr):
+    if is_numpy_array(arr):
+        return numpy
+    elif is_cupy_array(arr):
         return cupy
     else:
-        return numpy
+        raise ValueError(
+            "Only numpy and cupy arrays are supported"
+            f", but found {type(arr)} instead. If "
```

Comment:
I wonder if we want the `If [...] Thinc.`. This is true for a lot of exceptions in Thinc and I don't think we'd want these verbose errors.

It's up to the caller to decide whether to add more context to errors.

Also, if this fails somewhere, I am not sure how a downstream user knows whether it was or wasn't called directly.

Reply:
Ah, I missed that. Fine with me as well!

Reply:
It would definitely be nice to have the errors in Thinc structured more properly like we do in spaCy (in a different PR though). For this particular one, I don't mind so much and am fine either way, so you get the decisive vote, Ákos!